backend/apis/partners/update-partner.api.js

```diff
@@ -60,11 +62,13 @@ app.patch("/api/partners/:partnerId", (req, res) => {
       isActive: req.body.hasOwnProperty("isActive")
         ? req.body.isActive
         : partner.isActive,
+      phone: req.body.hasOwnProperty("phone") ? req.body.phone : partner.phone,
```

Comment:
I think we should transform the request phone, no?

```suggestion
      phone: req.body.hasOwnProperty("phone") ? requestPhone(req.body.phone) : partner.phone,
```